src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -691,6 +691,12 @@ public void runTests(boolean waitForApplicationUpdate, int messageOccurrences, T
                     error(e.getMessage(), e);
                 }
             }
+        } else {
```

Comment:
We can simplify/clarify this method a step by removing the wrapping `if (!forceSkipTests)`. The `else` logic can go into an `if (forceSkipTests)` placed at the beginning that does a single `return` after either info messages.

Reply:
Although this is true, I didn't want to change the existing logic when just adding a new log message.